addons/stock_account/models/stock_move.py

```diff
@@ -380,7 +380,7 @@ def _generate_valuation_lines_data(self, partner_id, qty, debit_value, credit_va
         # This method returns a dictionary to provide an easy extension hook to modify the valuation lines (see purchase for an example)
         self.ensure_one()
         debit_line_vals = {
-            'name': self.name,
+            'name': self.reference + ' - ' + self.product_id.name if self.reference else self.product_id.name,
```

Comment:
isn't `description` the same text? after this patch, do we have the same thing in `name` and `ref` on the am? maybe i'm confused

Reply:
yes, description is same. I was checking the same. And yes after this we will have same values in ref and name.
Please let me know if it is not correct.